pkg/k8s/k8s.go

```diff
@@ -38,6 +38,17 @@ const (
 	WatchVerb            = "watch"
 
 	ServiceDescriptorClaimVerb = "claim"
+
+	// From k8s.io/kubernetes/pkg/kubelet/apis/well_known_labels.go
```

Comment:
Replace it with a GitHub link? https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/well_known_labels.go

Also, there are GA labels in https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/well_known_labels.go#L36-L42
We should probably use them instead of "beta"? We currently run on k8s 1.12, not sure in which version were GA labels released? If they are missing in 1.12, let's create a separate follow-up ticket and leave a TODO here.

Reply:
According to the docs for 1.12 and 1.13, it's still the beta labels: https://v1-12.docs.kubernetes.io/docs/concepts/configuration/assign-pod-node/#interlude-built-in-node-labels

Reply:
And just to be sure, in the source: https://github.com/kubernetes/kubernetes/blob/v1.12.2/pkg/kubelet/apis/well_known_labels.go

Reply:
@halcyonCorsair update the link with a v1.12.2 one from GitHub?